ddtrace/internal/processor/trace.py

```diff
@@ -215,3 +216,33 @@ def shutdown(self, timeout):
         except ServiceStatusError:
             # It's possible the writer never got started in the first place :(
             pass
+
+
+@attr.s
+class SingleSpanSamplingProcessor(SpanProcessor):
```

Comment:
What do you think about just referring to "span sampling" rather than "single span sampling"?

```suggestion
class SpanSamplingProcessor(SpanProcessor):
```

Reply:
I think that makes sense.